libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts

```diff
@@ -42,6 +44,12 @@ export class ResultsComponent {
 
   @ViewChild('results', { static: false }) results!: ElementRef<HTMLDivElement>;
 
+  ngOnInit() {
+    setTimeout(() => {
+      this.canShowErrorMsg$.next(true);
+    }, 500);
+  }
+
```

Comment:
Unless I'm misunderstanding how this is implemented, it simply shows  an error message after 500ms.  Doesn't this mean if the backend service takes longer than 500ms to come back we'll end up with a flash of the error message followed by showing the error message?  Furthermore, doesn't this mean the error message will always show?  Shouldn't the display of the error be keyed to a failed or erroneous response from the estimate service?

Reply:
right. when i checked a few scenarios, i've received response in < 500ms. But, i found another approach by adding `error` clause which removed dependency on setTimeOut.

Reply:
Perfect.